### PR TITLE
fix several bugs in oncokb annotation scripts and improve code robustness

### PR DESCRIPTION
**Pull request by SRodenburg**
I ran into several issues while using these scripts. Can now also use local `*.json` files for gene symbol resolving.
Code improvements to handle edge cases in the data.

### CODE REVIEW

File: core/src/main/scripts/importer/importOncokbDiscreteCNA.py [hunk 1]
```diff
@@ -150,7 +156,11 @@ def get_cna_events(cna_file_path, server_url=None, alias_map=None):
         for sample_id in sample_ids:
             feature = {}
             feature[SAMPLE_ID] = sample_id
-            feature[ALTERATION] = int(line_elements[sample_indexes[sample_id]])
+            cna_value = line_elements[sample_indexes[sample_id]]
+            try:
+                feature[ALTERATION] = 0 if cna_value in cna_null_values else int(round(float(cna_value)))
+            except TypeError:
+                continue
```
Comment (SRodenburg) [inline core/src/main/scripts/importer/importOncokbDiscreteCNA.py:163]: this is prob the most important change

**Review by oplantalech — CHANGES_REQUESTED** [review]

File: core/src/main/scripts/importer/importOncokbDiscreteCNA.py (hunk 1 above)
Comment (SRodenburg) [reply core/src/main/scripts/importer/importOncokbDiscreteCNA.py:163]: just FYI, the cbio importer also rounds non-integer values (e.g. 1.5 -> 2) so this follows that same logic.
Values other than -2, 1, 1, 2 are skipped by default.

**Review by oplantalech — APPROVED** [review]

**Comment by SRodenburg** [comment]
@inodb can we merge this with tomorrow's release? Thanks :)

**Review by pvannierop — APPROVED** [review]
Nice addition!

**Comment by SRodenburg** [comment]
Any clue why the e2e tests are failing on Keycloak? seems unrelated to this PR

**Pull request merged**